lib/faye/redis.rb

```diff
@@ -175,6 +176,11 @@ def redis
         end
         register_connection_listeners('pubsub', @subscriber, "faye-server/#{@ns}/pubsub[#{Socket.gethostname}][#{Process.pid}]")
         register_connection_listeners('redis', connection, "faye-server/#{@ns}[#{Socket.gethostname}][#{Process.pid}]")
+        if inactivity_check[:enabled]
```

Comment:
Any reason to add a third `:enabled` option to the hash instead of just having `:trigger_secs` and `:response_timeout` and replacing this line by:
```suggestion
        if inactivity_check[:trigger_secs] && inactivity_check[:response_timeout]
```

I'm not saying that we can't keep it this way, but I'm wondering.

Reply:
I don't have a strong argument for keeping it.
I wanted to emphasize the fact that we activate something not simply update default value or something. I don't mind to remove it though if I do I'd opt for a `inactivity_check[:trigger_secs] || inactivity_check[:response_timeout]` condition if you are ok. My point is that if one option is not set (:trigger_secs or :response_timeout) `configure_inactivity_check` function will raise an `ArgumentError` and I think it's better to fail on misconfiguration than silently ignoring it.

WDYT?

Reply:
Makes sense, let's go with the `||`.